pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [
 python = "^3.12"
 datasets = "*"
 pandas = "*"
-litellm = "^1.52.3"
+litellm = {git = "https://github.com/BerriAI/litellm.git", rev = "litellm_dev_11_29_2024"}
```

Comment:
Just curious, what do we need from this version?

Reply:
It's to fix this issue: https://github.com/BerriAI/litellm/issues/6953

Reply:
Oh, interesting, thanks!